config/jobs/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/gcp-compute-persistent-disk-csi-driver-postsubmits.yaml

```diff
@@ -25,7 +25,7 @@ periodics:
       securityContext:
         privileged: true
   annotations:
-    testgrid-dashboards: sig-gcp-compute-persistent-disk-csi-driver
```

Comment:
Please don't move gcp-compute-persistent-disk-csi-driver jobs. @davidz627 is maintaining them and can help address any issues.

Reply:
I think these were "orphaned" due to being under `sig-gcp`. Can we move them under `sig-cloud-provider-gcp` or whatever the new name for this sig is? Also yes, I am still maintaining them